spring-cloud-bus/src/main/java/org/springframework/cloud/bus/jackson/BusJacksonAutoConfiguration.java

```diff
@@ -38,14 +38,15 @@
 public class BusJacksonAutoConfiguration {
 
 	@Bean
-	@ConditionalOnMissingBean(name="busJsonConverter")
+	@ConditionalOnMissingBean(name = "busJsonConverter")
```

Comment:
can you limit your change to not include formatting changes? makes it hard to see where the real change is.